src/EventStore.Core/TransactionLog/Chunks/TFChunkWriter.cs

```diff
@@ -43,35 +45,45 @@ public class TFChunkWriter : ITransactionFileWriter {
 		public bool Write(ILogRecord record, out long newPos) {
 			var result = _currentChunk.TryAppend(record);
 			if (result.Success)
-				_writerCheckpoint.Write(result.NewPosition + _currentChunk.ChunkHeader.ChunkStartPosition);
+				_nextRecordPosition = result.NewPosition + _currentChunk.ChunkHeader.ChunkStartPosition;
 			else
 				CompleteChunk(); // complete updates checkpoint internally
```

Comment:
i think the comment here is out of date now

Reply:
fixed :+1: